deploy/templates/gcs-manifests/gcs-csi.yml.j2

```diff
@@ -164,16 +164,21 @@ spec:
           args:
             - "--v=5"
             - "--csi-address=$(ADDRESS)"
+            - "--kubelet-registration-path=$(DRIVER_REG_SOCK_PATH)"
```

Comment:
The registration socket is `csi.sock` as I see it. If that is the case, ADDRESS already provides the path, so we could use that. We do not need an additional volume for the registration socket, and have a separate env variable it.

Reply:
there is an issue opened in driver-registrar to combine both flags, once it's done we can reconsider this and use only one volume
current implementation is done as  per CSI  doc [registrar-sidecar-configuratio](https://github.com/kubernetes-csi/docs/blob/master/book/src/Setup.md#registrar-sidecar-configuration)

Reply:
Thanks for the doc link. Makes it a little more clear.